### PR TITLE
Wazo 2266 remove get_queue_statistics sql function

### PR DESCRIPTION
**Pull request by vamenard**

### CODE REVIEW

File: alembic/versions/241501a5f8ba_remove_queue_info_function.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            cast (round(avg(hold_time)) as int)
+        FROM
+            queue_info
+        WHERE
+            queue_name = $1 and call_time_t > $2;
+        $$
+    LANGUAGE SQL;
+    """
+    op.get_bind().execute(query)
+    op.drop_table('queue_info')
```
Comment (fblackburn1) [inline alembic/versions/241501a5f8ba_remove_queue_info_function.py:66]: Please configure your IDE to run flake8 on every python file at each save or commit
W292 no newline at end of file

File: alembic/versions/241501a5f8ba_remove_queue_info_function.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            -- max_hold_time
+            max(hold_time),
+            -- mean_hold_time
+            cast (round(avg(hold_time)) as int)
+        FROM
+            queue_info
+        WHERE
+            queue_name = $1 and call_time_t > $2;
+        $$
+    LANGUAGE SQL;
```
Comment (fblackburn1) [inline alembic/versions/241501a5f8ba_remove_queue_info_function.py:63]: Please configure your IDE to run flake8 on every python file at each save or commit
W291 trailing whitespace

File: alembic/versions/241501a5f8ba_remove_queue_info_function.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            'queue_info',
+            Column('id', Integer, primary_key=True),
+            Column('call_time_t', Integer),
+            Column('queue_name', String(128), nullable=False, server_default=''),
+            Column('caller', String(80), nullable=False, server_default=''),
+            Column('caller_uniqueid', String(32), nullable=False, server_default=''),
+            Column('call_picker', String(80)),
+            Column('hold_time', Integer),
+            Column('talk_time', Integer),
+        )
```
Comment (fblackburn1) [inline alembic/versions/241501a5f8ba_remove_queue_info_function.py:34]: same comment about flake8
Fix indent

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

**Comment by bloom1** [comment]
Hello @vamenard!

Adding to what @fblackburn1 said, I don't think that it is necessary to have a working dowgrade in this case. Since this migration does not remove the table, it should not recreate it on downgrade. The queue statistics have not been present in Wazo since 18.03 (I think?) so they can be safely removed without a working dowgrade.

What do you think @fblackburn1?

**Comment by fblackburn1** [comment]
> Hello @vamenard!
> 
> Adding to what @fblackburn1 said, I don't think that it is necessary to have a working dowgrade in this case. Since this migration does not remove the table, it should not recreate it on downgrade. The queue statistics have not been present in Wazo since 18.03 (I think?) so they can be safely removed without a working dowgrade.
> 
> What do you think @fblackburn1?

Yep no objection. downgrade are more for development purpose, they are never used
And I agree in this case, running this downgrade AND the previous downgrade script, will raise a error because we try to create the same table twice

**Review by fblackburn1 — APPROVED** [review]
thanks, now https://jenkins.wazo.community/view/Failed/job/xivo-manage-db-check_migrations/
should be fixed

**Review by bloom1 — APPROVED** [review]

**Comment by bloom1** [comment]
regate

**Comment by bloom1** [comment]
recheck

**Pull request merged**